client/src/common.rs

```diff
@@ -204,11 +206,18 @@ pub mod block_hash {
             .display_order(order)
     }
 
-    pub(crate) fn get(matches: &ArgMatches) -> Option<BlockHash> {
+    pub(crate) fn get(matches: &ArgMatches) -> Option<BlockIdentifier> {
         matches.value_of(ARG_NAME).map(|hex_str| {
```

Comment:
We should rename `hex_str` to something less specific now, like e.g. `value`, since it's not always a hex string.

Reply:
Set as `value`